cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/aggregation/NGSIGenericColumnAggregator.java

```diff
@@ -128,6 +130,31 @@ public void aggregate(NGSIEvent event) {
                 aggregation.put(attrName + "_type", valuesType);
             } // if else
         } // for
+
+        if (event.getMappedCE() != null && isEnableNameMappings() && isEnableGeoParse()) {
+            mappedContextElement = event.getMappedCE();
+            mappedContextAttributes = mappedContextElement.getAttributes();
+            for (NotifyContextRequest.ContextAttribute mapedContextAttribute : mappedContextAttributes) {
+                if (mapedContextAttribute.getType() != null && (mapedContextAttribute.getType().equalsIgnoreCase("geo:json") || mapedContextAttribute.getType().equalsIgnoreCase("geo:point"))) {
```

Comment:
The equalsIgnoreCase() remembers me to the hack we discarded at the end (PR https://github.com/telefonicaid/fiware-cygnus/pull/1877 PR https://github.com/telefonicaid/fiware-cygnus/pull/1880).

Should we use equal() ? Or this is intentional?